environments/values-idfdemo.yaml

```diff
@@ -0,0 +1,23 @@
+name: "idfdemo"
+fqdn: "demo.lsst.cloud"
+appOfAppsName: "science-platform"
+butlerRepositoryIndex: "s3://butler-us-central1-repo-locations/data-dev-repos.yaml"
+butlerServerRepositories:
+  dp02: "https://demo.lsst.cloud/api/butler/repo/dp02/butler.yaml"
+gcp:
+  projectId: "science-platform-demo-9e05"
+  region: "us-central1"
+  clusterName: "science-platform-demo"
+onepassword:
+  connectUrl: "https://roundtable-dev.lsst.cloud/1password"
+  vaultTitle: "Phalanx data-demo.lsst.cloud"  # FIXME?  Wrong but consistent
```

Comment:
We'd need to change it though Chile IT, so probably no use for the comment here.